torch/csrc/jit/codegen/cuda/ir_utils.h

```diff
@@ -0,0 +1,101 @@
+#pragma once
+
+#include <torch/csrc/jit/codegen/cuda/type.h>
+
+namespace torch {
+namespace jit {
+namespace fuser {
+
+namespace ir_utils {
+
+#if 0
+template <
+    typename FilterType,
+    template <typename, typename...> typename Container,
+    typename ElementType>
+Container<FilterType*> filterVals(const Container<ElementType*>& container) {
+  Container<FilterType*> out;
+  for (auto& s : container) {
+    if (s->getValType() == FilterType::type) {
+      out.push_back(s->template as<FilterType>());
+    }
+  }
+  return out;
+}
+#else
+template <typename FilterType, typename InputIt>
```

Comment:
`InputIt` -> `IteratorType` or just `Iterator` ?

Reply:
Renamed to `Iterator`.